api/shape/startTimer.js

```diff
@@ -0,0 +1,10 @@
+import Shape from './Shape.js';
+
+export const starts = new Map();
+
+export const startTimer = (name) => (shape) => {
```

Comment:
This is a nice feature to have generally

Reply:
Yeah, I thought I had this already, but I couldn't find it. :)